libs/journey-uris/src/main/java/uk/gov/di/ipv/core/library/journeyuris/JourneyUris.java

```diff
@@ -28,4 +28,7 @@ private JourneyUris() {
     public static final String JOURNEY_TEMPORARILY_UNAVAILABLE_PATH =
             "/journey/temporarily-unavailable";
     public static final String JOURNEY_UNMET_PATH = "/journey/unmet";
+    public static final String JOURNEY_ALTERNATE_DOC_PATH = "/journey/alternate-doc";
+    public static final String JOURNEY_ALTERNATE_DOC_INVALID_DL_PATH =
```

Comment:
I can't see this being used anywhere, can it be dropped?

Reply:
used in test.
Also updated test names.